command/ca/provisionerbeta/add.go

```diff
@@ -101,13 +107,16 @@ func addCommand() cli.Command {
     : Uses the ACME protocol to create certificates.
 
     **X5C**
-    : Uses an X509 Certificate / private key pair to sign provisioning tokens.
+    : Uses an X509 certificate / private key pair to sign provisioning tokens.
 
     **K8SSA**
     : Uses Kubernetes Service Account tokens.
 
     **SSHPOP**
-    : Uses an SSH Certificate / private key pair to sign provisioning tokens.`},
+    : Uses an SSH certificate / private key pair to sign provisioning tokens.
+
+	**Nebula**
```

Comment:
Why is the indentation here different?

Reply:
Fixed with 7a151e0, I've changed tabs with spaces in that section and now it renders properly